assets/chat/js/menus/ChatUserInfoMenu.js

```diff
@@ -265,7 +265,7 @@ export default class ChatUserInfoMenu extends ChatMenuFloating {
       this.flairSubheader.style.display = 'none';
     }
 
-    const messageList = this.createMessages();
+    const messageList = this.createMessages(prettyNick);
```

Comment:
Is `prettyNick` just the user's display name?

Reply:
Yep, renamed it to `displayName` to be consistent.